### ISSUE TITLE
Shrink/Grow Height/Width not working

### ISSUE THREAD

**Issue by RaitaroH** [esjeon/krohnkite#41]
**Symptom**
I am not sure I understand what those shortcuts do. Are they supposed to be used for master window, slaves, floating windows? 

**How to reproduce** - not sure (I just installed the script today from git)
The shortcuts do nothing across the board. In this scenario I thought to simply stop sxkhd and try again, yet nothing. Then I thought to open konsole and invoke the shortcut with qdbus, still nothing. I also decided to test if just using the script with `make run` will say anything, yet nothing.
```
qdbus org.kde.kglobalaccel /component/kwin org.kde.kglobalaccel.Component.invokeShortcut'Krohnkite: Grow Height'
```

**Expected behavior**
Anything really.

**Environment**
Operating System: KDE neon 5.16
KDE Plasma Version: 5.16.0
KDE Frameworks Version: 5.59.0
Qt Version: 5.12.3
Kernel Version: 4.15.0-51-generic
OS Type: 64-bit
kwin 5.16.0

**Comment by esjeon** [comment]
Oh, my apologies for the confusion. I haven't implemented that part yet. You should use mouse to resize windows. Luckily, you can use Alt+RClick to resize windows, and this is also configurable in `Window Behavior`>`Window Actions`.

Sadly, the development this script is currently stalled, because I'm working on something quite distorted (which makes it difficult to multitask). I want to actively work on this project again, but it'll take a few more weeks until I stabilize my current project.

**Comment by RaitaroH** [comment]
Oh, I see. The options where there so I thought this was a bug. Would be great to see it implemented when you have the time.

Sad to see you don't have time for it because I think this is the only actually usable kwin tiling script. I can tell because I didn't uninstall it after 2 days 😛

**Comment by Delvien** [comment]
I cant adjust the height of windows with krohnkite running, only the width. Am I missing something?

**Comment by esjeon** [comment]
@Delvien , you are not missing anything, *yet*. Krohnkite was originally meant to be a clone of dwm, which doesn't have general window resizing features. But Tile layout currently supports free resizing using mouse, and I'm working on resizing using shortcuts.

**Comment by esjeon** [comment]
The shortcuts are implemented by d607c37f351df79753d8fd6acb46f1e4bb5b2f2d, but only for tiled windows, and for adjustable layouts (like tile, three column, quarter). I'll see how I can extend this feature for floating windows.

**Comment by esjeon** [comment]
Closing. v0.8 now supports moving and resizing floating windows w/ keyboard shortcuts.
